frontend/src/assets/css/variables.scss

```
@@ -1,18 +1,8 @@
 :root {
-  --white: #ffffff;
-  --shadow: 0 2px 4px #0000001a;
-  --black: #000000;
-  --gray-100: #e2e3e5;
-  --gray-200: #6c707a;
-  --gray-300: #302e36;
-  --blue-100: #ebf3ff;
-  --blue-200: #acb4d0;
-  --blue-300: #507ceb;
-  --violet-100: #e5e1ff;
-  --pink-200: #e1a7d5;
-  --red-300: #d51a52;
   --transparent: #00000000;
   --white-alpha-20: #ffffff33;
   --white-alpha-80: #ffffff80;
   --blue-alpha-30: #527deb4d;
+  --red-error: #d51a52;
+  --black-text: #302e36;
 }
```

return changes in this file back and use variables from it
```suggestion
:root {
  --white: #ffffff;
  --shadow: 0 2px 4px #0000001a;
  --black: #000000;
  --gray-100: #e2e3e5;
  --gray-200: #6c707a;
  --gray-300: #302e36;
  --blue-100: #ebf3ff;
  --blue-200: #acb4d0;
  --blue-300: #507ceb;
  --violet-100: #e5e1ff;
  --pink-200: #e1a7d5;
  --red-300: #d51a52;
  --transparent: #00000000;
  --white-alpha-20: #ffffff33;
  --white-alpha-80: #ffffff80;
  --blue-alpha-30: #527deb4d;
}
```